src/components/calcite-slider/calcite-slider.tsx

```diff
@@ -146,6 +146,9 @@ export class CalciteSlider implements LabelableComponent, FormComponent {
   disconnectedCallback(): void {
     disconnectLabel(this);
     disconnectForm(this);
+    document.removeEventListener("pointermove", this.dragUpdate);
```

Comment:
Since these lines are duplicated, can you extract the event removal to a private method to DRY things up?